fe/fe-core/src/main/java/org/apache/doris/analysis/InlineViewRef.java

```diff
@@ -266,7 +266,9 @@ public TupleDescriptor createTupleDescriptor(Analyzer analyzer) throws AnalysisE
             }
 
             columnSet.add(colAlias);
-            columnList.add(new Column(colAlias, selectItemExpr.getType().getPrimitiveType()));
+            // TODO: inlineView threat all column is nullable to make sure query results are correct
+            // we should judge column whether is nullable by selectItemExpr in the future
+            columnList.add(new Column(colAlias, selectItemExpr.getType().getPrimitiveType(), true));
```

Comment:
Unfortunately, we cannot directly obtain the information about whether slot desc is nullable or not from Expr.